cdap-common/src/main/java/co/cask/cdap/common/security/YarnTokenUtils.java

```diff
@@ -96,7 +96,7 @@ public static Credentials obtainToken(YarnConfiguration configuration, Credentia
           services.add(SecurityUtil.buildTokenService(YarnUtils.getRMAddress(configuration)).toString());
         }
 
-        Token<TokenIdentifier> token = ConverterUtils.convertFromYarn(rmDelegationToken, null);
+        Token<TokenIdentifier> token = ConverterUtils.convertFromYarn(rmDelegationToken, (InetSocketAddress) null);
```

Comment:
any reason this type casting is required?

Reply:
In later hadoop version, the the `convertFromYarn` mehtod has overload. Pass in `null` causes compilation error.

Reply:
I see ok.